src/main.rs

```diff
@@ -31,7 +31,7 @@ fn main() -> anyhow::Result<()> {
     abort_handler::setup();
     let command = Command::from_command_line()?;
     let api = CsesHttpApi::default();
-    let storage = FileStorage::default();
+    let mut storage = FileStorage::default();
```

Comment:
`mut` isn't necessary here, because the object is moved to via `resources` into `Ui`. So it will be mutated, just not in the scope of the `main` function.

CI seems to be currently failing because of this.